src/pseudopeople/configuration/validator.py

```diff
@@ -170,9 +175,10 @@ def _get_default_config_node(
 
 
 def _validate_possible_age_differences(
-    noise_type_config: Union[Dict, List],
+    noise_type_config: Any,
```

Comment:
Why Any?

Reply:
It's a validation function that fails if it actually gets the wrong type, but in principle the validation function works on any input.